packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -1991,6 +1991,9 @@ export class EntityService {
       createFieldArgs.relatedFieldDisplayName = !allowMultipleSelection
         ? entity.pluralDisplayName
         : entity.displayName;
+
+      createFieldArgs.relatedFieldAllowMultipleSelection =
+        !allowMultipleSelection;
```

Comment:
The name of the property seems to describe the opposite. Is it meant to be *NOT AllowMultipleSelection*?

Reply:
Yes
This is how it was before we changed it to use the args, so I just changed the args to have this value
![image](https://github.com/amplication/amplication/assets/39680385/9cca69af-369c-4f37-9722-9c70c5b7cd54)

Reply:
Fixed the naming for better understanding